platform/security/core/security-core-services/src/main/java/ddf/security/http/impl/HttpSessionFactory.java

```diff
@@ -19,11 +19,16 @@
 import ddf.security.common.audit.SecurityLogger;
 import ddf.security.http.SessionFactory;
 import java.nio.charset.StandardCharsets;
+import java.util.concurrent.TimeUnit;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpSession;
 
 public class HttpSessionFactory implements SessionFactory {
 
+  private static final int DEFAULT_EXPIRATION_TIME = 31;
```

Comment:
❓ Does this make more sense to be set to `30`? Correct me if I am wrong, but I believe that extra minute is for the warning, but based on the description added, it sounds like that should be a part of the expiration with "1 minute _remains_".

Reply:
Yeah you're right. I had just copied this from the other class that had this expiration logic. I'll put it to 30.